src/sidebars/star/star.less

```diff
@@ -15,34 +19,40 @@
   outline: 0;
   display: block;
   .confirmation-modal {
+    &.flipped{
+      box-shadow: -2px -2px 2px 0px @dark-background;
+    }
+    box-shadow: -2px 2px 2px 0px @dark-background;
```

Comment:
Is that really massively better? So this would become something like `.shadowMixin(@shadow-position:(-2px,-2px), @blur: (2px, 0px), @colour: @dark-background)` or `.shadowMixin((-2px,-2px), (2px, 0px), @dark-background)`